renderer/src/components/RebrandBanner.tsx

```diff
@@ -0,0 +1,36 @@
+import Text from './Text'
+import { useState } from 'react'
+import CloseIcon from 'src/assets/img/icons/close.svg?react'
+import { openExternalURL } from 'src/lib/station-config'
+import { useLocation } from 'react-router'
+import { ROUTES } from 'src/lib/routes'
+
+const RebrandBanner = () => {
+  const { pathname } = useLocation()
+  const [visible, setVisible] = useState<boolean>(true)
```

Comment:
This state isn't persisted, which means that on every restart of the app the banner will reappear. I think that should be ok, but wanted to double check if that's what you intended. If not, we can persist this state, which I can add to this PR if you want to

Reply:
I'm happy for it to reappear after a refresh as a reminder. I decided to leave it this way intentionally, when I was thinking of whether I needed to do more. I don't think it will affect the user experience and it will remind the operator of upcoming changes. This banner and its message may easily change within the next few weeks too.